package.json

```diff
@@ -8,15 +8,15 @@
     "test": "vitest --watch=false"
   },
   "devDependencies": {
-    "@astrojs/mdx": "^0.16.0",
-    "@astrojs/tailwind": "^3.0.1",
-    "@astrojs/vue": "^2.0.1",
+    "@astrojs/mdx": "^1.0.3",
+    "@astrojs/tailwind": "^5.0.0",
+    "@astrojs/vue": "^3.0.0",
     "@babel/eslint-parser": "^7.14.7",
     "@babel/eslint-plugin": "^7.14.5",
     "@fontsource/inter": "^4.5.15",
     "@headlessui/vue": "^1.6.0",
     "@tailwindcss/typography": "^0.5.8",
-    "astro": "^2.0.11",
+    "astro": "^3.0.12",
```

Comment:
[Upgraded Astro and our integrations as per the migration guide](https://docs.astro.build/en/guides/upgrade-to/v3/#upgrade-astro)